python/mxnet/gluon/contrib/estimator/estimator.py

```diff
@@ -222,28 +230,30 @@ def evaluate(self,
     def fit(self, train_data,
```

Comment:
Any thought on how to make resuming from the checkpoint easier? With the current code, if I want to train 90 epochs and the training failed at epoch 30, the steps I need to do to resume training are:
- create net
- call net.load_parameters, with the correct .params filename
- create trainer
- call trainer.load_states, with the correct .states filename
- create an estimator
- call fit with epoch = 60? 

Since we know the naming scheme of the checkpoint, we can do all these automatically by loading the last checkpoint?